backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/qrcodes/service/HUQRCodeGenerateForExistingHUsResult.java

```diff
@@ -0,0 +1,36 @@
+package de.metas.handlingunits.qrcodes.service;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.ImmutableSetMultimap;
+import com.google.common.collect.SetMultimap;
+import de.metas.handlingunits.HuId;
+import de.metas.handlingunits.qrcodes.model.HUQRCode;
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.ToString;
+import org.adempiere.exceptions.AdempiereException;
+
+@EqualsAndHashCode
+@ToString
+public class HUQRCodeGenerateForExistingHUsResult
+{
+	private final ImmutableSetMultimap<HuId, HUQRCode> huQRCodes;
+
+	HUQRCodeGenerateForExistingHUsResult(@NonNull final SetMultimap<HuId, HUQRCode> huQRCodes)
+	{
+		this.huQRCodes = ImmutableSetMultimap.copyOf(huQRCodes);
+	}
+
+	public ImmutableSetMultimap<HuId, HUQRCode> toSetMultimap() {return huQRCodes;}
+
```

Comment:
consider introducing `List<HUQRCode> toList() { .... }` because i saw u also needed it in ur processes